### ISSUE TITLE
import as causes syntax error

### ISSUE THREAD

**Issue by objectProgrammer** [bjansen/pebble-intellij#52]
https://pebbletemplates.io/wiki/tag/import/

```
{% import "something.peb" as something %}
```

causes a syntax error.

**Comment by objectProgrammer** [comment]
Also worth to note that obviously usages of the import name causes unknown variable warning.

**Comment by bjansen** [comment]
Man, Pebble is such a tricky language because it doesn't really define keywords, and allows this kind of syntax:

```twig
{% from "templates/macros/macro.peb" import import as import, as as as %}
```

Which means my ANTLR parser can't define `as` and `import` as keywords, and I have to highlight keywords manually based on their position.

**Comment by objectProgrammer** [comment]
Yeah, I love pebble but sometimes it is really loose. And oh wow, I don't think anybody would understand that statement without focusing on it for like 5 minutes.

**Comment by bjansen** [comment]
Already better :) 
<img width="420" alt="image" src="https://user-images.githubusercontent.com/281528/101685684-59a61b00-3a68-11eb-8ec5-1b0a3e16fd12.png">

**Comment by bjansen** [comment]
And of course your example works too:

<img width="310" alt="image" src="https://user-images.githubusercontent.com/281528/101685822-80645180-3a68-11eb-8684-7453eb4838b3.png">

**Comment by bjansen** [comment]
@objectProgrammer could you try this build and let me know if it works on your files please?

[pebble-intellij.zip](https://github.com/bjansen/pebble-intellij/files/5668765/pebble-intellij.zip)

Macro names and aliases are still highlighted as unknown, but that will likely be fixed in #19

**Comment by objectProgrammer** [comment]
@bjansen Yes it works, thank you! The syntax error is gone :)
